src/platform/utilities/api/index.js

```diff
@@ -50,8 +50,15 @@ function isJson(response) {
  * @param {Function} **(DEPRECATED)** success - Callback to execute after successfully resolving
  * the initial fetch request.
  * @param {Function} **(DEPRECATED)** error - Callback to execute if the fetch fails to resolve.
+ * TODO
  */
-export function apiRequest(resource, optionalSettings = {}, success, error) {
+export function apiRequest(
+  resource,
+  optionalSettings = {},
+  success,
+  error,
```

Comment:
Unfortunately, I cannot remove these deprecated args yet, because they are still used in other files. E.g. https://github.com/department-of-veterans-affairs/vets-website/blob/9ab4ce64d40646fd0934095919715dc71e287193/src/applications/post-911-gib-status/actions/post-911-gib-status.js#L17.